src/test/java/com/github/tomakehurst/wiremock/matching/NegativeContainsPatternTest.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2016-2022 Thomas Akehurst
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.tomakehurst.wiremock.matching;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.github.tomakehurst.wiremock.client.WireMock;
+import org.junit.jupiter.api.Test;
+
+public class NegativeContainsPatternTest {
```

Comment:
A 3rd test case would be good showing what happens when the actual value is null, as can happen when e.g. the request header being compared is absent.

Reply:
Yes, good point. I don't know why I forgot about adding this.

Thank you for all those comments. I don't know what are the contribution rules for commits, therefore I just amend my commit with fixes applied to it.